oauth2client/service_account.py

```diff
@@ -607,7 +604,8 @@ def authorize(self, http):
             h = credentials.authorize(h)
         """
         request_orig = http.request
-        request_auth = super(_JWTAccessCredentials, self).authorize(http).request
+        request_auth = super(_JWTAccessCredentials,
```

Comment:
"Eww, yuck" to the whole expression on the right side of this assignment statement, but: can we line-wrap immediately after `super(` instead so that `_JWTAccessCredentials, self` is not broken across lines?